hello-jni/app/build.gradle

```diff
@@ -26,12 +26,31 @@ android {
             path "src/main/cpp/CMakeLists.txt"
         }
     }
-    // Documentation purpose: app/src/main/jni is reserved for deprecated native build system;
-    // Steer away from this directory by DELETING it for ndk-build/cmake + android studio usage.
-    // Otherwise you need to
-    //    add android.useDeprecatedNdk=true to gradle.properties
-    //    enable the following line to silence android studio complaint
-    // sourceSets.main.jni.srcDirs = []
+    productFlavors {
+        // Create product flavors with different formats
+        // all other samples will use just ONE simple format
+        arm7 {
+            // in the future, ndk.abiFilter might also work
+            ndkConfig.abiFilter 'armeabi-v7a'
+        }
+        arm8 {
+            ndkConfig.abiFilters 'arm64-v8a'
+        }
+        arm {
+            ndk {
+                abiFilter 'armeabi'
```

Comment:
Weird. Why is this one different?
Also all of these seem kind of redundant. Is there no simpler way to just treat each ABI as a product flavor?